db/structure.sql

```diff
@@ -9,48 +9,20 @@ SET xmloption = content;
 SET client_min_messages = warning;
 SET row_security = off;
 
---
--- Name: plpgsql; Type: EXTENSION; Schema: -; Owner: -
---
-
-CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
```

Comment:
This is extra?

Reply:
I think its disallowed, I'll double check

Reply:
It actually works fine, re-including it